sqlglot/dialects/nuodb.py

```diff
@@ -1,14 +1,21 @@
 from __future__ import annotations
 
 from sqlglot import exp, generator, parser, tokens, transforms
-from sqlglot.dialects.dialect import (Dialect, no_properties_sql,no_comment_column_constraint_sql)
+from sqlglot.dialects.dialect import (Dialect,no_comment_column_constraint_sql)
 from sqlglot.tokens import Tokenizer, TokenType, Token
 
 
 global schema_name
 schema_name = None
 
 
+
+def no_properties_sql(self: generator.Generator, expression: exp.Properties) -> str:
+    if isinstance(expression, exp.PartitionedByProperty):
+        return ""
```

Comment:
This function always returns "" regardless.  Is that the intention?

Reply:
no_properties_sql just states whether the property is supported or unsupported by the dialect.
So, in this case, I dont want NuoDB unsupport Partition so, excluding Partition other properties which are stated are unsupported